.travis.yml

```diff
@@ -1,7 +1,7 @@
 ---
 language: node_js
 node_js:
-  - "6"
+  - "8"
```

Comment:
@samselikoff `package.json` still claims that this project is compatible with Node 6. was this an intentional change?

Reply:
Isn't this part of the default blueprint now?

Reply:
yes, and in https://blog.emberjs.com/2019/05/21/ember-3-10-released.html we specifically recommended to release a new major version when dropping support for Node 6